cosmo_tester/resources/terraform/openstack-vm.tf.template

```diff
@@ -75,6 +75,36 @@ resource "openstack_compute_secgroup_v2" "security_group" {
     ip_protocol = "tcp"
     cidr = "${openstack_networking_subnet_v2.subnet.cidr}"
   }
```

Comment:
We tested and without those rules it didn't work. But it was about a month ago. I spoke with Omer about it and he said to me that this is a bug in design.  Something changed?